docs/docs/codes/modifiers.mdx

```diff
@@ -30,13 +30,15 @@ These act like any other key code.
 
 ### Modifier Functions
 
-These functions take the form: `XX(code)`
-
-- They _apply_ the modifier to another _code_.
-  - e.g. `&kp LS(A)` = `LEFT_SHIFT`+`A`
-    - aka a _shifted_ (capitalized) **A**.
-  - The _code_ and the _modifier_ are handled _as one_.
-- They can be _combined_:
-  - e.g. `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
-- Some codes _include_ modifiers in their definition.
-  - e.g. `DOLLAR` = `LS(N4)`
+Modifier functions add one or more modifier to another code. These functions take the form: `XX(code)`
+
+- Modifier functions apply the modifier to another code:
+  - `&kp LS(A)` = `LEFT_SHIFT`+`A` (a capitalized **A**).
+- They can be combined:
+  - `&kp LC(RA(B))` = `LEFT_CONTROL`+`RIGHT_ALT`+`B`
+- Some basic codes codes already include a modifier function in their definition:
+  - `DOLLAR` = `LS(NUMBER_4)`
+- There are left- and right-handed versions of each modifier:
```

Comment:
Do we need this with the table above?

Reply:
I think it's nice to have them in one place. I immediately thought "ok which modifier functions exist?" when I read this chapter in the docs.

Reply:
Perhaps add a note to see the table above as it also provides more information around each modifier.